packages/opentelemetry-instrumentation-alephalpha/examples/basic_tracing.py

```
@@ -0,0 +1,30 @@
+
+# 4. Perform a Completion Request
+request = CompletionRequest(
+    prompt=Prompt.from_text("Explain LLM Observability in one sentence."),
+    maximum_tokens=30,
+)
+response = client.complete(request)
+
+# 5. Print Response Output
+if response.completions:
+    print("\n--- Response ---")
+    print(response.completions[0].completion)
```

Ruff (0.16.1)

[warning] No newline at end of file

Add trailing newline

(W292)